### PR TITLE
Add some triage role collaborators

### PR DESCRIPTION
**Pull request by kezhenxu94**
Add @kezhenxu94 @zhongjiajie @caishunfeng as triage role collaborators

The triage role allows people to assign, edit, and close issues and pull requests, without giving them write access to the code.

### CODE REVIEW

**Review by zhongjiajie — APPROVED** [review]
LGTM

**Review by CalvinKirs — APPROVED** [review]
LGTM

**Comment by wu-sheng** [comment]
Interesting setting. As a member of PMC, why don't we discuss about committer promotion?
I will do this on private mail list. Please hold this PR for a while.

**Comment by kezhenxu94** [comment]
> Interesting setting. As a member of PMC, why don't we discuss about committer promotion?
> I will do this on private mail list. Please hold this PR for a while.

This is not committer promotion, it grants external people to triage issues, without write permissions to the repo

**Comment by wu-sheng** [comment]
My feeling is strange, let me submit discussion to PMC members.

**Comment by wu-sheng** [comment]
All about writing permission is based on trust policy, which should not set the bar too high.

**Pull request closed without merging**
